### PR TITLE
add a card preparer for dataset xlsum

### PR DESCRIPTION
**Pull request by dafnapension**

### CODE REVIEW

**Comment by dafnapension** [comment]
this script can run all the way through only after pull request 288 is merged

File: prepare/cards/xlsum.py [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+            metrics=["metrics.rouge"],
+        ),
+        templates=TemplatesList(
+            [
+                InputOutputTemplate(input_format="{text}", output_format="{target}"),
+            ]
+        ),
+    )
+    if lang == langs[0]:
+        test_card(card, debug=True)
```
Comment (yoavkatz) [inline prepare/cards/xlsum.py:37]: set debug = False by default to speed up regressions.

File: prepare/cards/xlsum.py [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+langs = configs
+
+for lang in langs:
+    card = TaskCard(
+        loader=LoadHF(path="GEM/xlsum", name=lang),
+        preprocess_steps=[
+            # SplitRandomMix({"train": "train[95%]", "validation": "train[5%]", "test": "validation"}),
+            # CopyFields(field_to_field=[["answers/text", "answer"]], use_query=True),
+        ],
+        task=FormTask(
```
Comment (yoavkatz) [inline prepare/cards/xlsum.py:25]: In general, we should move to a summarization task that is defined outside the card.  We can setup a meeting to discuss.

**Review by yoavkatz — CHANGES_REQUESTED** [review]
Changes in the comments.

**Comment by dafnapension** [comment]
open  PR #301 instead, and recommitted  with signature

**Pull request closed without merging**